docs/development.md

```diff
@@ -16,6 +16,34 @@ By default, this script will deploy a `postgresql` instance, the worker and the
 
 You can skip deployment and basic settings if you use the script.
 
+### MacOS Users
+
+If you are using MacOS to develop for OpenScanHub you will need to set up a different environment:
+
+1. You will need to install the latest Bash version using brew:
+    `brew install bash`
+2. Locate the new package:
+   `which bash` -> `/opt/homebrew/bin/bash`
+3. Run it:
+    `/opt/homebrew/bin/bash`
+4. In the new console, you can run the scripts form the [Using scripts](#using-scripts) section.
```

Comment:
Hmm, AFAIK, all scripts have a shebang to execute `bash` through `env` so this step should not be necessary.  Could please check?

Reply:
I had to do this because if you just execute the script directly, the terminal you are using will be used instead of the recently installed. In this case, zsh and the default macOS terminal (GNU Bash version 3.2) doesn't support the "mapfile" function and doesn't let the script run. That's why I added these extra steps

Reply:
Just to prevent confusion, by terminal you mean a shell, right?  There shouldn't be any difference between the output of `which` and the binary that will be used because our shebangs invoke `bash` through `env` (e.g. #!/usr/bin/env bash) so the step 3 still seems redundant to me.  

Nevertheless, I believe that there is no real reason to use `mapfile` and, thus, force macOS users to install a newer version of `bash` from Homebrew.  I'll try to simplify the script a bit a I'll let you know when it's ready to be tested.  Thank you!